src/components/teamPhotos.js

```diff
@@ -53,18 +58,99 @@ const ProfileRole = styled.span`
 `
 
 const ProfileImg = styled(Img)`
-  border-radius: 50%;
+  transition: filter ease-in 200ms;
+`
+
+const ProfileImgWrapper = styled.div`
+  position: relative;
+
+  @media (min-width: 1000px) {
+    &:hover .profileImg {
+      filter: blur(1.5px);
+    }
+    &:hover .web {
```

Comment:
`.web` refers to the `IconWrapper` that is inside the `ProfileImgWrapper` and `.mobile` refers to the `IconWrapper` that is below the `ProfileRole`. The classNames seemed necessary because on mobile I need to make `.web { display: none; }` and on web, I need to make `.mobile { display: none; }`.

This existed before I introduced the `react-responsive` library. Is there an obviously more simple way to do this? I couldn't think of a way to only have 1 instance of `IconWrapper`.